server/src/main/java/org/opensearch/cluster/service/ClusterManagerThrottlingStats.java

```diff
@@ -8,18 +8,29 @@
 
 package org.opensearch.cluster.service;
 
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.common.io.stream.Writeable;
 import org.opensearch.common.metrics.CounterMetric;
+import org.opensearch.common.xcontent.ToXContent;
+import org.opensearch.common.xcontent.ToXContentFragment;
+import org.opensearch.common.xcontent.XContentBuilder;
 
+import java.io.IOException;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
 /**
  * Contains stats of Cluster Manager Task Throttling.
  * It stores the total cumulative count of throttled tasks per task type.
  */
-public class ClusterManagerThrottlingStats implements ClusterManagerTaskThrottlerListener {
+public class ClusterManagerThrottlingStats implements ClusterManagerTaskThrottlerListener, Writeable, ToXContentFragment {
 
-    private Map<String, CounterMetric> throttledTasksCount = new ConcurrentHashMap<>();
+    private Map<String, CounterMetric> throttledTasksCount;
+
+    public ClusterManagerThrottlingStats() {
```

Comment:
We might need additional metrics on throttle_time

Reply:
Can you please elaborate more on this? 
What we are expecting out of throttle_time metric?

Reply:
time since we last throttled

Reply:
Time since cluster manager performed has done throttling for any task? Or for different task type as well? 

I think we can achieve it, as we can maintain time as well when we have performed last throttling on cluster manager node. 

I am just wondering how this additional metric will help?